tasks/blacklight.rake

```diff
@@ -25,42 +30,31 @@ namespace :blacklight do
     Rake::Task["spec"].invoke
   end
 
-  desc "Create the test rails app"
```

Comment:
Unclear why this is needed; looks like cruft from the pre-EngineCart days. Found no references to it anywhere on the open web, so I removed it.

Reply:
I think this is needed because blacklight is a rails engine too and to test that functionality the engine_cart is used.

Reply:
Removing this doesn't seem to have caused any ill effects, FWIW.